root/artist/split.tt

```diff
@@ -36,6 +36,15 @@
          <div id="artist-credit-editor"></div>
        </fieldset>
 
+       [% IF collaborators.size %]
+         <h3>[% add_colon(l('Collaborators on this artist')) %]</h3>
```

Comment:
I wonder if it'd be better to show this first, before the AC editor.

Reply:
I think so, made the change.